doc/develop/west/install.rst

```diff
@@ -51,7 +51,7 @@ Enabling shell completion
 West currently supports shell completion in the following combinations of
 platform and shell:
 
-* Linux: bash
+* Linux: bash, zsh, fish
 * macOS: bash
 * Windows: not available
```

Comment:
Can we remove this whole section? One can get Bash on Windows with MinGW32

Reply:
Yes, that is true. I agree with @nguyenmthien.